xcube/webapi/controllers/catalogue.py

```diff
@@ -316,6 +323,28 @@ def get_dataset(ctx: ServiceContext,
     return dataset_dict
 
 
+def get_bbox_geometry(dataset_bounds: Tuple[float, float, float, float],
```

Comment:
I could well imagine this method as a utility function someplace else. Also, a dedicated unit test would have been nice.

Reply:
Done.

Reply:
Right, once it is used a second time, we'll do so. Currently it is called unconditionally, therefore test is not required w.r.t. coverage.